io_directory_example/settings/lens_system3_config.yml

```diff
@@ -19,20 +19,36 @@ model:
     - 'SERSIC_ELLIPSE'
 
   source_light:
+    - 'SERSIC_ELLIPSE'
     - 'SHAPELETS'
 
  # point_source: ['LENSED_POSITION'] # 'LENSED_POSITION', 'SOURCE_POSITION'
 
 
 lens_option:
   centroid_init: [0.04, -0.04]
+  gaussian_prior:
+    0 :
+      - ['gamma', 2.11, 0.03]
+      - ['theta_E', 1.11, 0.13]
 
 lens_light_option:
   fix: { 0: { 'n_sersic': 4. }, 1: { 'n_sersic': 3.5 } }
+  gaussian_prior:
+    0:
+      - [ 'R_sersic', 0.21, 0.15 ]
   #fix: {0: {'n_sersic': 4.}} # fix n_sersic to 4. for 1st light profile
 
 source_light_option:
   n_max: [2, 4] # array with n_max for each source light profile
+  gaussian_prior:
+    0:
+      - [ 'R_sersic', 0.21, 0.15 ]
+
+point_source_option:
+  gaussian_prior:
+    0:
+      - [ 'ra_image', 0.21, 0.15 ]
```

Comment:
Remove space from in front of `'ra_image'`.

Reply:
Is this issue resolved?

Reply:
I think so as I removed the gaussian_prior for the point source option in the setting file